cfy_manager/components/globals.py

```diff
@@ -68,14 +68,12 @@ def _set_external_port_and_protocol():
     config[MANAGER]['external_rest_protocol'] = external_rest_protocol
 
 
-def _set_rabbitmq_config():
-    if not config[RABBITMQ].get('broker_cert_path'):
-        config[RABBITMQ]['broker_cert_path'] = constants.CA_CERT_PATH
```

Comment:
The relevant part of this configuration is set by the rabbitmq component installer now.